index.js

```diff
@@ -16,46 +20,71 @@ var defaultOptions = {
 module.exports = {
   name: 'ember-bootstrap',
 
-  included: function included(app) {
+  included(app) {
```

Comment:
I'm good with that. I just want to make sure we're not invalidating our backwards compatibility targets.